### PR TITLE
Add uyuni-storage-setup-server and uyuni-storage-setup-proxy packages to server/proxy deployment

### PR DESCRIPTION
**Pull request by raulillo82**
# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Add 2 extra packages to be installed in the deployment of Server/Proxy (one for each):  uyuni-storage-setup-[server|proxy]

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only: Uyuni
* Does this PR need to be backported? NO
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed): Not needed for 4.3 or 5.0, as this is Uyuni only.

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Added uyuni-storage-setup-[server|proxy] packages to Server/Proxy deployment
```
Comment (juliogonzalez) [inline CHANGELOG.md:1]: I am not sure if this should be:
```suggestion
- Added uyuni-storage-setup-server package to Server deployment and
  uyuni-storage-setup-proxy to Proxy deployment
```
But I will leave it to @keichwa to decide.

**Review by juliogonzalez — APPROVED** [review]
LGTM, other than my minor comment (Karl to decide)

File: CHANGELOG.md (hunk 1 above)
Comment (raulillo82) [reply CHANGELOG.md:1]: Yes, I agree this should be changed, but will wait for Karl before committing the suggestion.

Comment (keichwa) [reply CHANGELOG.md]: suggestions sounds good.  Added to batch commission.  If you are faster, also fine.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+- Added uyuni-storage-setup-server package to Server deployment and
+  uyuni-storage-setup-proxy to Proxy deployment
```
Comment (keichwa) [inline CHANGELOG.md:2]:
```suggestion
- Added uyuni-storage-setup-server package to server deployment and
  uyuni-storage-setup-proxy to proxy deployment in Installation and Upgrade
  Guide
```
FYI, we only capitalize server or proxy if "{productname}" or something similar precedes it.  It once was a Lana decision and I still support it ;)

**Review by keichwa — APPROVED** [review]
cosmetic things.
command lines are long, but that's the way it is.

**Pull request merged**